packages/axios-extension/README.adoc

```diff
@@ -44,14 +44,32 @@ const provider = createForAbap({
     baseURL: 'https://sap.example',
     params: { 'sap-client': client }
 });
-const service = provider.layeredRepository();
+const service = provider.getLayeredRepository();
 await service.deploy('./dist/my-variant-webapp.zip', {
     namespace: 'apps/my.base.app/appVariants/customer.variant/',
     package: 'MY_PACKAGE',
     transport: 'ABC123'
 });
 ```
 
+#### ADT Service
```

Comment:
Thanks for this Shen. The way consumers add new ADT services will be by extending `AdtService` (that is why we expose the method `getAdtService()`). Could you add a quick note about how to do that?